cytodynamics-nucleus/src/main/java/com/linkedin/cytodynamics/nucleus/LoaderBuilder.java

```diff
@@ -54,8 +56,25 @@ public LoaderBuilder withOriginRestriction(OriginRestriction originRestriction)
     return this;
   }
 
-  public LoaderBuilder addParentRelationship(ParentRelationship parentRelationship) {
-    this.parentRelationships.add(parentRelationship);
+  /**
+   * Set the parent relationship for the loader. This will be checked first when trying to load a class.
+   *
+   * @param parentRelationship primary {@link ParentRelationship} to use for the loader
+   */
+  public LoaderBuilder withParentRelationship(ParentRelationship parentRelationship) {
+    this.parentRelationship = parentRelationship;
+    return this;
+  }
+
+  /**
+   * Add a fallback delegate classloader for the loader. If an entity cannot be loaded through the primary delegate
+   * relationship, then the loader will attempt to find it in the fallback(s).
+   * The fallback delegates will be used in the order that they are added to this builder.
+   *
+   * @param fallbackDelegate a fallback {@link ParentRelationship} to use for the loader
+   */
+  public LoaderBuilder addFallbackDelegate(ClassLoader fallbackDelegate) {
```

Comment:
Good question.
Having the single parent was so that it was clear what the main parent classloader was. Using the vanilla class loaders was to simplify the API/implementation, and the use case I have doesn't need the checks. 
I was going back and forth on using `ParentRelationship`s for the fallbacks. It does add nice flexibility. Since you recognized this use case as well, I will add it back in, since it does impact the API.